src/account-info/AccountDetails.js

```diff
@@ -1,7 +1,8 @@
+
 import React, { Component } from 'react';
 import { StyleSheet, Text, View } from 'react-native';
 
-import { Avatar, Button, UserStatusIndicator } from '../common';
+import { Avatar, ZButton, UserStatusIndicator } from '../common';
```

Comment:
I know that React Native now has a button component, but I don't think we need to rename this. We can always use `import { Button as ZButton }` if necessary